js/pro/cex.js

```diff
@@ -0,0 +1,1229 @@
+        let baseId = this.safeString (ticker, 'symbol1');
+        if (baseId === undefined) {
+            baseId = this.safeString (pair, 0);
+        }
+        let quoteId = this.safeString (ticker, 'symbol2');
+        if (quoteId === undefined) {
+            quoteId = this.safeString (pair, 1);
+        }
+        const base = this.safeCurrencyCode (baseId);
+        const quote = this.safeCurrencyCode (quoteId);
+        const symbol = base + '/' + quote;
+        let timestamp = this.safeInteger (ticker, 'timestamp');
+        if (timestamp !== undefined) {
+            timestamp = timestamp * 1000;
+        }
+        return this.safeTicker ({
+            'symbol': symbol,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'high': this.safeString (ticker, 'high'),
+            'low': this.safeString (ticker, 'low'),
+            'bid': this.safeString (ticker, 'bid'),
+            'bidVolume': undefined,
+            'ask': this.safeString (ticker, 'ask'),
+            'askVolume': undefined,
+            'vwap': undefined,
+            'open': this.safeString (ticker, 'open24'),
+            'close': undefined,
+            'last': this.safeString2 (ticker, 'price', 'last'),
+            'previousClose': undefined,
+            'change': this.safeString (ticker, 'priceChange'),
+            'percentage': this.safeString (ticker, 'priceChangePercentage'),
+            'average': undefined,
+            'baseVolume': undefined,
+            'quoteVolume': this.safeString (ticker, 'volume'),
+            'info': ticker,
+        }, market);
+    }
+
+    async watchOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        /**
+         * @method
+         * @name cex#watchOrders
+         * @description get the list of orders associated with the user. Note: In CEX.IO system, orders can be present in trade engine or in archive database. There can be time periods (~2 seconds or more), when order is done/canceled, but still not moved to archive database. That means, you cannot see it using calls: archived-orders/open-orders.
+         * @see https://docs.cex.io/#ws-api-open-orders
+         * @param {string} symbol unified symbol of the market to fetch trades for
+         * @param {int|undefined} since timestamp in ms of the earliest trade to fetch
+         * @param {int|undefined} limit the maximum amount of trades to fetch
+         * @param {object} params extra parameters specific to the cex api endpoint
+         * @returns {[object]} a list of [trade structures]{@link https://docs.ccxt.com/en/latest/manual.html?#public-trades}
+         */
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' watchOrders requires a symbol argument');
+        }
+        await this.loadMarkets ();
+        await this.authenticate (params);
+        const url = this.urls['api']['ws'];
```

Comment:
@pcriadoperez can you check what is going on with `watchOrders` on the python side? 
![image](https://user-images.githubusercontent.com/43336371/202490974-41527b9b-82eb-4944-9fc6-f71755b069a1.png)